### PR TITLE
ID Localization for "Get Shell Running Container" from Task.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR is addressing `/docs/tasks/debug-application-cluster/get-shell-running-container/` as mentioned in #20111

### CODE REVIEW

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+title: Dapatkan _Shell_ Untuk Masuk ke Kontainer yang Sedang Berjalan
```
Comment (sftim) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:2]: You can't use HTML or Markdown in titles
```suggestion
title: Dapatkan Shell Untuk Masuk ke Kontainer yang Sedang Berjalan
```

Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:2]: Sorry... my bad! Thanks @sftim.

Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:2]:
```suggestion
title: Mendapatkan Shell Untuk Masuk ke Container yang Sedang Berjalan
```

Let's don't use italic, and leave Container be it as it's an API Object.

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+Halaman ini menunjukkan bagaimana cara menggunakan `kubectl exec` untuk
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:8]:
```suggestion
Laman ini menunjukkan bagaimana cara menggunakan `kubectl exec` untuk 
```

https://kbbi.kemdikbud.go.id/entri/laman

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
+---
+title: Dapatkan _Shell_ Untuk Masuk ke Kontainer yang Sedang Berjalan
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+Halaman ini menunjukkan bagaimana cara menggunakan `kubectl exec` untuk 
+mendapatkan _shell_ untuk masuk ke dalam kontainer yang sedang berjalan.
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:9]:
```suggestion
mendapatkan _shell_ untuk masuk ke dalam Container yang sedang berjalan.
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+
+{{% capture steps %}}
+
+## Mendapatkan sebuah _shell_ untuk masuk ke sebuah kontainer
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:23]:
```suggestion
## Mendapatkan sebuah _shell_ untuk masuk ke sebuah Container
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+Dalam latihan ini, kamu perlu membuat Pod yang hanya memiliki satu kontainer saja. Kontainer
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:25]:
```suggestion
Dalam latihan ini, kamu perlu membuat Pod yang hanya memiliki satu Container saja. Container
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+tersebut menjalankan _image_ nginx. Berikut ini adalah _file_ konfigurasi untuk Pod tersebut:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:26]:
```suggestion
tersebut menjalankan _image_ nginx. Berikut ini adalah berkas konfigurasi untuk Pod tersebut:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+{{< codenew file="application/shell-demo.yaml" >}}
+
+Buatlah Pod:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:30]:
```suggestion
Buatlah Pod tersebut:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/shell-demo.yaml
+```
+
+Pastikan bahwa kontainer dalam Pod berjalan:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:36]:
```suggestion
Pastikan bahwa Container dalam Pod berjalan:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+
+```shell
+kubectl get pod shell-demo
+```
+
+Dapatkan _shell_ untuk masuk ke dalam kontainer:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:42]:
```suggestion
Dapatkan _shell_ untuk masuk ke dalam Container:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+
+```shell
+kubectl exec -it shell-demo -- /bin/bash
+```
+
+{{< note >}}
+
+Simbol tanda hubung ganda "--" digunakan untuk memisahkan antara argumen perintah yang ingin kamu eksekusi pada kontainer dan argumen dari kubectl itu sendiri.
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:50]:
```suggestion
Simbol tanda hubung ganda "--" digunakan untuk memisahkan antara argumen perintah yang ingin kamu eksekusi pada Container dan argumen dari kubectl itu sendiri.
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+
+{{< /note >}}
+
+Dalam _shell_ kamu, perlihatkan isi dari direktori _root_:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:54]:
```suggestion
Di dalam _shell_ kamu, perlihatkan isi dari direktori _root_:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+
+```shell
+root@shell-demo:/# ls /
+```
+
+Dalam _shell_ kamu, cobalah dengan perintah-perintah yang lainnya. Berikut ini beberapa contoh diantaranya:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:60]:
```suggestion
Di dalam _shell_ kamu, cobalah perintah-perintah lainnya. Berikut beberapa contohnya:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+root@shell-demo:/# apt-get install -y lsof
+root@shell-demo:/# lsof
+root@shell-demo:/# apt-get install -y procps
+root@shell-demo:/# ps aux
+root@shell-demo:/# ps aux | grep nginx
+```
+
+## Menulis halaman utama (_root_) untuk nginx
+
+Lihat kembali pada _file_ konfigurasi untuk Pod kamu. Pod
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:78]:
```suggestion
Lihat kembali berkas konfigurasi untuk Pod kamu. Pod
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+memiliki _volume_ `emptyDir`, dan kontainer melakukan _mounting untuk _volume_ tersebut
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:79]:
```suggestion
memiliki volume `emptyDir`, dan Container melakukan pemasangan (_mounting_) untuk volume tersebut
```

Comment (phanama) [reply content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:79]: kbbi.kemdikbud.go.id/entri/volume

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+pada `/usr/share/nginx/html`.
+
+Pada _shell_ kamu, buatlah _file_ `index.html` dalam direktori `/usr/share/nginx/html`:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:82]:
```suggestion
Pada _shell_ kamu, buatlah berkas `index.html` di dalam direktori `/usr/share/nginx/html`:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 16]
```diff
@@ -0,0 +1,147 @@
...
+
+Pada _shell_ kamu, kirimkan sebuah permintaan (_request_) GET ke server nginx.
+
+```shell
+root@shell-demo:/# apt-get update
+root@shell-demo:/# apt-get install curl
+root@shell-demo:/# curl localhost
+```
+
+Tampilan akan menunjukkan teks yang kamu tulis pada _file_ `index.html`.
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:96]:
```suggestion
Keluarannya akan menunjukkan teks yang kamu tulis pada berkas `index.html`.
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+
+```shell
+Hello shell demo
+```
+
+Setelah kamu selesai dengan _shell_ kamu, ketiklah `exit`.
+
+## Menjalankan perintah individu dalam sebuah kontainer
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:104]:
```suggestion
## Menjalankan perintah individu di dalam sebuah Container
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+
+Pada jendela (_window_) perintah biasa, bukan pada _shell_ kamu dalam kontainer,
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:106]:
```suggestion
Pada jendela (_window_) perintah biasa, bukan pada _shell_ kamu di dalam Container,
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+lihatlah daftar lingkungan variabel pada kontainer yang sedang berjalan:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:107]:
```suggestion
lihatlah daftar variabel lingkungan (_environment variable_) pada Container yang sedang berjalan:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+
+```shell
+kubectl exec shell-demo env
+```
+
+Cobalah dengan menjalankan perintah lainnya. Berikut ini beberapa contoh diantaranya:
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:113]:
```suggestion
Cobalah dengan menjalankan perintah lainnya. Berikut beberapa contohnya:
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 21]
```diff
@@ -0,0 +1,147 @@
...
+kubectl exec shell-demo ps aux
+kubectl exec shell-demo ls /
+kubectl exec shell-demo cat /proc/1/mounts
+```
+
+{{% /capture %}}
+
+{{% capture discussion %}}
+
+## Membuka sebuah _shell_ ketika sebuah Pod memiliki lebih dari satu kontainer
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:125]:
```suggestion
## Membuka sebuah _shell_ ketika sebuah Pod memiliki lebih dari satu Container
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 22]
```diff
@@ -0,0 +1,147 @@
...
+
+Jika sebuah Pod memiliki lebih dari satu kontainer, gunakanlah `--container` atau `-c` untuk
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:127]:
```suggestion
Jika sebuah Pod memiliki lebih dari satu Container, gunakanlah `--container` atau `-c` untuk
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 23]
```diff
@@ -0,0 +1,147 @@
...
+menentukan kontainer yang dimaksud dalam perintah `kubectl exec`. Sebagai contoh,
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:128]:
```suggestion
menentukan Container yang dimaksud pada perintah `kubectl exec`. Sebagai contoh,
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+misalkan kamu memiliki Pod yang bernama my-pod, dan Pod tersebut memiliki dua kontainer
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:129]:
```suggestion
misalkan kamu memiliki Pod yang bernama my-pod, dan Pod tersebut memiliki dua Container
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+yang bernama main-app dan helper-app. Perintah berikut ini akan membuka sebuah
+_shell_ ke kontainer dengan nama main-app.
```
Comment (phanama) [inline content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:131]:
```suggestion
_shell_ ke Container dengan nama main-app.
```

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md (hunk 1 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:2]: Ok! sorry my bad :)

File: content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md (hunk 14 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/get-shell-running-container.md:79]: Ok!

**Comment by ariscahyadi** [comment]
> Sorry for the long response. @ariscahyadi
> Thanks for the nice translation! I've reviewed with some comments, let's address them!

Thanks, @phanama ... I have addressed all of them!

**Comment by girikuncoro** [comment]
@ariscahyadi thanks for your contribution, this is well localized!

/lgtm
/approve

**Pull request merged**